dts/bindings/usb/uac2/zephyr,uac2-audio-streaming.yaml

```diff
@@ -78,3 +78,12 @@ properties:
     type: int
     description: |
       Number of effectively used bits in audio subslot.
+
+  polling-interval:
+    type: int
+    description: |
+      Input or output endpoint polling interval (bInterval).
+      This value influences how frequently the Host will poll the USB Endpoint.
+      For full-speed, the polling rate is 1000us * (2**(bInterval-1))
+      For high-speed, the polling rate is 125us * (2**(bInterval-1))
+      Defaults to 1, and ranges to 16 inclusive.
```

Comment:
It does not look right to me. I think it should be `polling-perion-us` describing polling period in microseconds. Below in UAC2 it should be converted to speed specific endpoint bInterval values using USB_FS_ISO_EP_INTERVAL/USB_HS_ISO_EP_INTERVAL macros.